packages/frontend/src/components/accounts/ledger/SetupLedger.js

```diff
@@ -93,7 +103,7 @@ const SetupLedger = (props) => {
                             return;
                         }
 
-                        await dispatch(createNewAccount({ accountId, fundingOptions, recoveryMethod: 'ledger', publicKey, recaptchaToken })).unwrap();
+                        await dispatch(createNewAccount({ accountId, fundingOptions, recoveryMethod: 'ledger', publicKey, recaptchaToken, path: customLedgerHdPath })).unwrap();
```

Comment:
If we're only dispatching `checkAndHideLedgerModal` in case of `createNewAccount`, what happens in the other scenarios?